PokemonTabletopAdventures_v3/PTA3.css

```diff
@@ -771,6 +816,15 @@ label.sheet-afflictions-text {
   width: 70px;
 }
 
+.charsheet input.sheet-custom-move-name[type="text"] {
```

Comment:
`custom-form-name` perhaps?

Also, this would be the ideal place to put the padding/margin rule from my previous comment.